### PR TITLE
Update README to add 32 bit support in 0.7

### PR DESCRIPTION
**Pull request by danmosemsft**
Also change .NET Core 2.0 - >2.1 as 2.0 is now out of support.

### CODE REVIEW

**Comment by danmosemsft** [comment]
Do we want to be more specific about `partial functionality` eg., `all functionality except TensorFlow and LightGBM`

**Comment by shauheen** [comment]
Thanks @danmosemsft

File: README.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ Along with these ML capabilities, this first release of ML.NET also brings the f
...
 
-ML.NET runs on Windows, Linux, and macOS - any platform where x64 [.NET Core](https://github.com/dotnet/core) or later is available. In addition, .NET Framework on Windows x64 is also supported.
+ML.NET runs on Windows, Linux, and macOS using [.NET Core](https://github.com/dotnet/core), or Windows using .NET Framework. 64 bit is supported on all platforms. 32 bit is supported on Windows, except for TensorFlow and LightGBM related functionality.
 
-The current release is 0.6. Check out the [release notes](docs/release-notes/0.6/release-0.6.md) to see what's new.
+The current release is 0.7. Check out the [release notes](docs/release-notes/0.7/release-0.7.md) to see what's new.
 
-First, ensure you have installed [.NET Core 2.0](https://www.microsoft.com/net/learn/get-started) or later. ML.NET also works on the .NET Framework. Note that ML.NET currently must run in a 64-bit process.
+First, ensure you have installed [.NET Core 2.1](https://www.microsoft.com/net/learn/get-started) or later. ML.NET also works on the .NET Framework 4.6.2 or later, but 4.7.2 or later is recommended.
```
Comment (eerhardt) [inline README.md]: `4.6.2 or later` => `4.6.1 or later`.  netstandard2.0 works on `4.6.1`.

File: README.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ Along with these ML capabilities, this first release of ML.NET also brings the f
 
 [![NuGet Status](https://img.shields.io/nuget/v/Microsoft.ML.svg?style=flat)](https://www.nuget.org/packages/Microsoft.ML/)
 
-ML.NET runs on Windows, Linux, and macOS - any platform where x64 [.NET Core](https://github.com/dotnet/core) or later is available. In addition, .NET Framework on Windows x64 is also supported.
+ML.NET runs on Windows, Linux, and macOS using [.NET Core](https://github.com/dotnet/core), or Windows using .NET Framework. 64 bit is supported on all platforms. 32 bit is supported on Windows, except for TensorFlow and LightGBM related functionality.
```
Comment (eerhardt) [inline README.md]: `except for TensorFlow and LightGBM` => `except for TensorFlow, LightGBM, and Onnx`

**Pull request merged**